lib/ditcher.js

```diff
@@ -172,13 +174,23 @@ function generateReturn(document, type) {
           retDoc.fields = {};
           i = 1;
         }
-        retDoc.fields[field] = document[field];
+        if(translateObjectsEnabled()) {
+          retDoc.fields[field] = translateDateObject(document[field]);
+        } else {
+          retDoc.fields[field] = document[field];
+        }
       }
     }
   }
   return retDoc;
 }
 
+function translateObjectsEnabled() {
+  return process.env.FH_SERIALISE_DATES === "yes"
```

Comment:
I would check for both "yes" and "true".

Reply:
ok, I'll change that and do a case-insensitive compare. (However I'd normally prefer to be prescriptive with a switch, saying set it to this value to enable, rather than the "truthy" approach of saying, set it to something "positive" to activate the functionality e.g. "on" & "1"